management/publishSettings.go

```diff
@@ -7,7 +7,7 @@ import (
 	"fmt"
 	"io/ioutil"
 
-	"github.com/Azure/go-pkcs12"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
```

Comment:
See https://github.com/Azure/azure-sdk-for-go/blob/master/arm/compute/client.go#L22 for example. This is what godeps does to your import paths. This is not wrong, IMHO.